CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.6.2 (2021-04-28)
+
+- Fixed the description list component to properly provide the `MutationObserverService`. [#215](https://github.com/blackbaud/skyux-layout/pull/215)
```

Comment:
Not really worth worrying, but we don't really need `the` in front of the code snippet here. Superminor, so I'd just leave it as is rather than have the checks run again. But JFYI for future reference, if we have a code snippet like that where there it isn't acting as an adjective before a noun (such as "property" or "service" or so on), then we generally don't include an article before it.